azurelinuxagent/common/agent_supported_feature.py

```diff
@@ -58,15 +59,35 @@ def __init__(self):
                                                   supported=_MultiConfigFeature.__SUPPORTED)
 
 
+class _ETPFeature(AgentSupportedFeature):
```

Comment:
Keeping in tune with the Supported Features, added a new feature for ETP and using this everywhere to check if it is supported or not